xpra/platform/posix/keyboard.py

```diff
@@ -51,7 +51,7 @@ def init_vars(self) -> None:
             self.__dbus_helper = None
             log.info(f"gnome input sources requires dbus: {e}")
         else:
-            self._dbus_gnome_shell_eval_ism(".inputSources", self._store_input_sources)
+            self._dbus_gnome_shell_ism("List", [], self._store_input_sources)
```

Comment:
This breaks backwards compatibility for any installation that has not been update to gnome-shell 41, doesn't it?

Reply:
No, it is backward-compatible. I tested it on `v3.32.2` too.
In older versions, both methods (**Eval** and our new extension) will work equally.
In newer versions, only the extension works.